### ISSUE TITLE
NVDA does not read Skype contact status for Windows 10

### ISSUE THREAD

**Issue by gdata1** [nvaccess/nvda#7612]
### Steps to reproduce:
> A list of the steps you take to demonstrate the problem. 
>
> Example:
>
> 1. Open skype For Windows 10
> 2. Browse to Contact List
> 3. Place the cursor on any contact
> 4. Nvda doesn't read the status.

> Please also remember to attach a zip of any files required to reproduce the issue.

### Expected behavior:
> Tell us what should happen.NVDA should read the user's online offline status, etc.

### Actual behavior:
> Tell us what happens instead.Nvda reads only the name of the contact.

### System configuration:
NVDA version:
> next-14432,821b31df

NVDA Installed or portable instol

Other information:
> Example: Running in a VM

Windows version:
> Windows 10 Version 1709 Build 16288.1

skype,, 12.4.676.0,

### Other questions:

Does the issue still occur after restarting your PC?
yes
Have you tried any other versions of NVDA?
> No

**Comment by josephsl** [comment]
Hi, and what happens if you disable add-ons? Thanks.

**Comment by zstanecic** [comment]
Hi
This happens with add-ons disabled, too.
Using windows 10 build 1703
Best,
ZS

Wysłane z aplikacji Poczta dla Windows 10

Od: Joseph Lee
Wysłano: poniedziałek, 18 września 2017 16:05
Do: nvaccess/nvda
DW: Subscribed
Temat: Re: [nvaccess/nvda] NVDA does not read Skype contact status forWindows 10 (#7612)

Hi, and what happens if you disable add-ons? Thanks.

**Comment by josephsl** [comment]
Hi, looks like I need to monitor what’s up unless there’s no suitable UIA/event combo. CC @MichaelDCurran

**Comment by zstanecic** [comment]
Hi, will the debug log with the events tracking in win 10 apps help you?


Wysłane z aplikacji Poczta dla Windows 10

Od: Joseph Lee
Wysłano: poniedziałek, 18 września 2017 17:25
Do: nvaccess/nvda
DW: zstanecic; Comment
Temat: Re: [nvaccess/nvda] NVDA does not read Skype contact status forWindows 10 (#7612)

Hi, looks like I need to monitor what’s up unless there’s no suitable UIA/event combo. CC @MichaelDCurran

**Comment by josephsl** [comment]
Hi, yes. Thanks.

**Comment by zstanecic** [comment]
OOps,
Sorry for my inconsistency:
1. This thing works with narrator
2. nVDA stable withot addons reports the status

Wysłane z aplikacji Poczta dla Windows 10

Od: Joseph Lee
Wysłano: poniedziałek, 18 września 2017 17:25
Do: nvaccess/nvda
DW: zstanecic; Comment
Temat: Re: [nvaccess/nvda] NVDA does not read Skype contact status forWindows 10 (#7612)

Hi, looks like I need to monitor what’s up unless there’s no suitable UIA/event combo. CC @MichaelDCurran

**Comment by zstanecic** [comment]
Hi
Here we are
DEBUG - external:globalPlugins.wintenObjs.GlobalPlugin.uiaDebugLogging (17:46:10.282):
W10: UIA object name: Adaleta, Niepodłączony, event: elementSelected, app module: <'skypeapp' (appName u'skypeapp', process ID 5096) at address 5392110>, automation Id: Contact, class name: ListViewItem
DEBUG - NVDAObjects.NVDAObject._get_placeholder (17:46:10.282):
Potential unimplemented child class: <NVDAObjects.UIA.ListItem object at 0x05392D10>
IO - braille.BrailleBuffer.update (17:46:10.288):
Braille regions text: [u'Skype ', u'Skype okn ', u'ContactsDialogPivot zak\u0142 ', u'Kontakty wbr zak\u0142adka 1 z 2 ', u'Lista kontakt\xf3w lst ', u'Kontakty na Skypie grp 1 z 1 lv 1 ', u'Adaleta, Niepod\u0142\u0105czony 6 z 42 lv 2']
IO - braille.BrailleHandler.update (17:46:10.288):
Braille window dots: 17 145 1 123 15 2345 1 2 - 13457 24 15 1234 135 145 126 16 14 1356 135 1345 13456 - 1248 - 1356 - 1458 128 - 123 1236 - 128
IO - inputCore.InputManager.executeGesture (17:46:12.549):
Input: kb(desktop):alt+f4
DEBUGWARNING - NVDAObjects.IAccessible.IAccessible._get_IAccessibleRole (17:46:12.572):
accRole failed: (-2147024809, 'Parametr jest niepoprawny.', (None, None, None, 0, None))
IO - speech.speak (17:46:12.687):
Speaking [u'Privatny - TeamTalk 5 Classic v. 5.2.3  dialog  Statyczny tekst']
DEBUG - NVDAObjects.NVDAObject._get_placeholder (17:46:12.687):
Potential unimplemented child class: <NVDAObjects.Dynamic_DialogIAccessibleWindowNVDAObject object at 0x052BCDD0>
IO - braille.BrailleBuffer.update (17:46:12.687):
Braille regions text: [u'Privatny - TeamTalk 5 Classic v. 5.2.3 dlg Statyczny tekst']
IO - braille.BrailleHandler.update (17:46:12.687):
Braille window dots: 12347 1235 24 1236 1 2345 1345 13456 - 36 - 23457 15 1 134 23457 1 123 13 - 158 - 147 123 1 234 234 24 14 - 1236 3 - 158 3 128 3 148 -
IO - inputCore.InputManager.executeGesture (17:46:13.739):
Input: kb(desktop):windows+d
IO - speech.speak (17:46:13.877):
Speaking [u'FolderView  lista']
IO - speech.speak (17:46:13.880):
Speaking [u'Google Chrome  6 z 21']
DEBUG - NVDAObjects.NVDAObject._get_placeholder (17:46:13.881):
Potential unimplemented child class: <NVDAObjects.IAccessible.sysListView32.List object at 0x053828D0>
DEBUG - NVDAObjects.NVDAObject._get_placeholder (17:46:13.881):
Potential unimplemented child class: <NVDAObjects.IAccessible.sysListView32.ListItem object at 0x05392E30>
IO - braille.BrailleBuffer.update (17:46:13.882):
Braille regions text: [u'FolderView lst ', u'Google Chrome 6 z 21']
IO - braille.BrailleHandler.update (17:46:13.884):
Braille window dots: 1247 135 123 145 15 1235 12367 24 15 2456 - 123 234 2345 - 12457 135 135 1245 123 15 - 147 125 1235 135 134 15 - 1248 - 1356 - 128 18
IO - inputCore.InputManager.executeGesture (17:46:13.979):
Input: kb(desktop):tab
IO - speech.speak (17:46:14.023):
Speaking [u'Start  przycisk']
DEBUG - NVDAObjects.NVDAObject._get_placeholder (17:46:14.023):
Potential unimplemented child class: <NVDAObjects.Dynamic_StartButtonButtonIAccessibleWindowNVDAObject object at 0x05392E70>
IO - braille.BrailleBuffer.update (17:46:14.025):


Wysłane z aplikacji Poczta dla Windows 10

Od: Joseph Lee
Wysłano: poniedziałek, 18 września 2017 17:43
Do: nvaccess/nvda
DW: zstanecic; Comment
Temat: Re: [nvaccess/nvda] NVDA does not read Skype contact status forWindows 10 (#7612)

Hi, yes. Thanks.

**Comment by josephsl** [comment]
Hi, STR please.

**Comment by zstanecic** [comment]
1. Open skype,
2. 2. Press alt+2
3. Notice, that the dialog appearance changed
4. Press tab two Times until you reach the list,
5. After that navigate to any contact you have on the list.
6. Used nvda 2017.3 stable
Note: win ten apps latest version was used.


Wysłane z aplikacji Poczta dla Windows 10

Od: Joseph Lee
Wysłano: poniedziałek, 18 września 2017 17:53
Do: nvaccess/nvda
DW: zstanecic; Comment
Temat: Re: [nvaccess/nvda] NVDA does not read Skype contact status forWindows 10 (#7612)

Hi, STR please.

**Comment by josephsl** [comment]
Hi, in short, changes were made to that list item, so a feedback to MS is in order. Thanks.

**Comment by gdata1** [comment]
Hello. I sent feedback to NS to give someone a problem.
https://aka.ms/Obqhyj

**Comment by Adriani90** [comment]
Testing with NVDA 2019.1.1 and Skype UWP 8.42 and Skype electron for desktop 8.43, this issue is not reproducible for me. @gdata1, @zstanecic, @k-kolev1985 could anyone of you please test if you can reproduce it?

**Comment by gdata1** [comment]
I don't have this problem

**Comment by Adriani90** [comment]
Closing as works for me. Thanks for reporting back. If someone still has this problem, please comment and we can reopen the discussion.
